opal/mca/common/ofi/common_ofi.c

```diff
@@ -190,6 +190,7 @@ int opal_common_ofi_export_memory_monitor(void)
             free(opal_common_ofi_monitor);
         }
 
+        mca_base_framework_close(&opal_memory_base_framework);
```

Comment:
Would there be an issue of closing this if it hasn't been opened? Line 141 is a goto err before this gets opened

Reply:
good point it probably shouldn't hurt but i'll add in a sentinel.

Reply:
@wckzhang check now